bottom-bar/src/main/java/com/roughike/bottombar/BottomBar.java

```diff
@@ -442,7 +466,7 @@ public void selectTabAtPosition(int position) {
      * Select a tab at the specified position.
      *
      * @param position the position to select.
-     * @param animate should the tab change be animated or not.
+     * @param animate  should the tab change be animated or not.
```

Comment:
the code formatter tries to set the description of the param where it thinks is best, not a manual change